public/main.js

```diff
@@ -2,23 +2,27 @@ window.main = (() => {
     const pane_config = [
         {
             label: 'Tic Tac Toe',
+            name: 'tic-tac-toe',
```

Comment:
we aren't doing anything with these yet but they could come useful later for styling or dynamically switching tabs with code.